src/arrayMethodIndexOf.js

```diff
@@ -5,8 +5,22 @@
  *
  */
 function applyCustomIndexOf() {
-  [].__proto__.indexOf2 = function(searchElement, fromIndex) {
-    // write code here
+  [].__proto__.indexOf2 = function(searchElement, fromIndex = 0) {
+    // if (fromIndex < 0)  {
+    //   fromIndex = this.length + fromIndex;
+    // }
+
+    // if (fromIndex < 0) {
+    //   return -1;
+    // }
+
```

Comment:
I added this comment, because I think these two operators are necessary for checking the index '-', but tests pass in both cases.